apps/studio/pages/project/[ref]/auth/providers.tsx

```diff
@@ -1,25 +1,72 @@
 import { PermissionAction } from '@supabase/shared-types/out/constants'
 
-import { AuthProvidersForm } from 'components/interfaces/Auth'
+import { AuthProvidersForm, BasicAuthSettingsForm } from 'components/interfaces/Auth'
 import AuthLayout from 'components/layouts/AuthLayout/AuthLayout'
-import { FormsContainer } from 'components/ui/Forms/FormsContainer'
-import NoPermission from 'components/ui/NoPermission'
 import { useCheckPermissions, usePermissionsLoaded } from 'hooks/misc/useCheckPermissions'
+import { useParams } from 'common'
+import { useCurrentPath } from 'hooks/misc/useCurrentPath'
+import Link from 'next/link'
+import { NavMenu, NavMenuItem } from 'ui'
+import {
+  ScaffoldContainer,
+  ScaffoldDescription,
+  ScaffoldDivider,
+  ScaffoldHeader,
+  ScaffoldTitle,
+} from 'components/layouts/Scaffold'
+import NoPermission from 'components/ui/NoPermission'
 import type { NextPageWithLayout } from 'types'
 
 const PageLayout: NextPageWithLayout = () => {
   const canReadAuthSettings = useCheckPermissions(PermissionAction.READ, 'custom_config_gotrue')
   const isPermissionsLoaded = usePermissionsLoaded()
+  const { ref } = useParams()
+  const currentPath = useCurrentPath()
+
+  const navMenuItems = [
+    {
+      label: 'Supabase auth',
+      href: `/project/${ref}/auth/providers`,
+    },
+    {
+      label: 'Third Party auth',
+      href: `/project/${ref}/auth/third-party`,
+    },
+  ]
 
   if (isPermissionsLoaded && !canReadAuthSettings) {
     return <NoPermission isFullPage resourceText="access your project's auth provider settings" />
-  } else {
-    return (
-      <FormsContainer className="w-full">
-        <AuthProvidersForm />
-      </FormsContainer>
-    )
   }
+
+  return (
+    <div>
+      <ScaffoldHeader className="pb-0">
+        <ScaffoldContainer id="auth-page-top">
+          <ScaffoldTitle>Sign in / up</ScaffoldTitle>
```

Comment:
Should we capitalize the title here? "Sign In / Up" - to match the side nav too
![image](https://github.com/user-attachments/assets/7544815d-0fc7-4f92-a84d-9766601c5762)